include/test/battle.h

```diff
@@ -819,6 +902,22 @@ enum { TURN_CLOSED, TURN_OPEN, TURN_CLOSING };
 #define TURN for (OpenTurn(__LINE__); gBattleTestRunnerState->data.turnState == TURN_OPEN; CloseTurn(__LINE__))
 
 #define MOVE(battler, ...) Move(__LINE__, battler, (struct MoveContext) { APPEND_TRUE(__VA_ARGS__) })
+
+#define EXPECTED_MOVE(battler, ...) ExpectedMove(__LINE__, battler, (struct MoveContext) { APPEND_TRUE(__VA_ARGS__) })
+#define NOT_EXPECTED_MOVE(battler, _move) ExpectedMove(__LINE__, battler, (struct MoveContext) { .move = _move, .explicitMove = TRUE, .notExpected = TRUE, .explicitNotExpected = TRUE, })
```

Comment:
I agree with what you said, though I had my reasons for using `MoveContext`. `allowed` is of no use, but I wanted to have an option for potential AI tests to specify things like critical hit, secondary effect and mega evolution. 

As for combining them, I couldn't really think of a way to do that without losing the ability to specify target(Since `EXPECT_MOVES` can't specify the target). I'm kinda blocked with what I have atm. 
Though if you have any ideas, maybe it could be changed 👀